app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+                  >${recentProjects[5].workStack[2]}</a
+                >
+              </li>
+            </ul>
+          `;
+    }
+
+    if (desktopProjectDiv.innerHTML !== '') {
+      descktopProjectsWrapperDiv.appendChild(desktopProjectDiv);
+    }
+  }
+}
+
+// portfolio end
+
+// popup start
+
+const modal = document.querySelector('.project-details-popup');
+const overlay = document.querySelector('.overlay');
+const openModalBtn = document.querySelectorAll('.pop');
+
+overlay.classList.add('hidden');
+const popupContainer = document.getElementById('project-details-popup');
+popupContainer.innerHTML = ` <div class="project-header">
+        <h3 class="project-title-popup">Multi-Post Stories</h3>
+        <i class="fa fa-close popup-close" id="close-x"></i>
+      </div>
+      <div>
+        <ul class="project-technology-popup">
+          <li>
+            <a
+              href="https://html.com/"
+              class="project-btn-pop project-btn-1-pop active-btn-effect hover-btn-effect"
+            >
+              HTML</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://getbootstrap.com/"
+              class="project-btn-pop project-btn-2 active-btn-effect hover-btn-effect"
+              >Bootstrap</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://www.ruby-lang.org/en/"
+              class="project-btn-pop project-btn-3 active-btn-effect hover-btn-effect"
+              >Ruby on Rails</a
+            >
+          </li>
+        </ul>
+      </div>
+      <div class="project-more">
+        <div>
+          <img
+            src="images/Snapshoot Portfolio.png"
+            alt=""
+            class="popup-project-img"
+          />
+        </div>
+
+        <div class="popup-project-more-details">
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries, but also the leap into electronic typesetting,
+            remaining essent
+          </p>
+          <ul class="popup-project-buttons">
+            <li class="first-child">
+              <a href=""
+                >See live<img
+                  src="images/live-Icon.png"
+                  alt="Live-icon"
+                  class="popup-project-button-icon-1"
+              /></a>
+            </li>
+            <li class="last-child">
+              <a href=""
+                >See source<img
+                  src="images/Icon -GitHub-white.png"
+                  alt="Live-icon"
+                  class="popup-project-button-icon-2"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>`;
```

Comment:
🎉👍 Thank you for your effort in adding the popup functionality to your project! 🌟 This is a significant step forward in making your project more interactive and user-friendly. However, there's a crucial aspect that needs attention.

- [ ] 📝💡 The popups should be populated dynamically based on the project details. Currently, the popup content remains the same for all projects, which is not aligned with the project requirements. 💻🔧 To resolve this, you'll need to dynamically generate the content for each project's popup. When a user clicks on the "Show Project" button, the popup should display the relevant information for that specific project, such as title, description, technologies used, live demo link, and project source code link. 👩‍💻💡 To achieve this, you can pass the project details as arguments to the function that creates the popup and then populate the modal with the corresponding information.

🚀📝 By making these adjustments, your project will be more in line with the design requirements and provide a better user experience by displaying accurate and relevant information for each project. 👏 Keep up the good work and happy coding! You're doing great! 💪🎉🚀